examples/loki/nurf_data.py

```diff
@@ -102,120 +102,205 @@ def nurf_file_creator(loki_file, path_to_loki_file, data):
 
     # open the file and append
     with h5py.File(loki_file, 'a') as hf:
+                
+        #comment on names
+        #UV/FL_Background is the dark
+        #UV/FL_Intensity0 is the reference
+        #UV/FL_Spectra is the sample
         
+        # image_key: number of frames (nFrames) given indirectly as part of the shape of the arrays 
+        # TODO: keep in mind what happens if multiple dark or reference frames are taken
+        
+        # remove axis=2 of length one from this array
+        data['UV_spectra']=np.squeeze(data['UV_spectra'], axis=2)  #this removes the third axis in this array, TODO: needs later to be verified with real data from hardware
+        
+        # assemble all spectra in one variable
+        uv_all_data=np.row_stack((data['UV_spectra'],data['UV_background'], data['UV_intensity0']))
+      
+        
+        # assemble image_key 
+        # #TODO: needs later to be verified with real data from hardware
+        uv_nb_spectra=np.shape(data['UV_spectra'])[0]
+        uv_ik_spectra=np.zeros((1,uv_nb_spectra))  #interperation here: 0 for sample (in comparison to projections)
+      
+        # find out how many nFrames each item (sample, dark, reference) has
+        if data['UV_background'].ndim==1:
+            uv_nb_darks=1
+        else: 
+            uv_nb_darks=np.shape(data['UV_background'])[1]  #TODO: needs to be verified with real data from Judith's setup
+   
+        uv_ik_dark=2* np.ones((1,uv_nb_darks)) 
+        
+        if data['UV_intensity0'].ndim==1:
+            uv_nb_ref=1
+        else:
+            uv_nb_ref=np.shape(data['UV_intensity0'])[1]  #TODO: needs to be verified with real data from Judith's setup
+        uv_ik_ref=4*np.ones((1,uv_nb_ref))  #new image key: 4 for reference
```

Comment:
Try something like `np.full(uv_nb_ref, 4)`.

Reply:
I tried this out, but it returns an array. Until the next commit, I keep my version.

Reply:
`ones` also returns an array... isn't that what you want?

Reply:
I just noticed something else, the `uv_integration_time` has only one number, but I would think each spectra should have an integration time. Should I fix it or we leave this for later?

Reply:
Do we want the `x_spectrum_key` in axis 0 or axis 1, please? Can we easily append to them, when they are `(9,)` and` (4,)` as it is now?

Reply:
- Each spectrum should have an integration time.
- Spectrum key should be 1D, i.e., has only one axis.

Reply:
Done with the integration time for each spectrum. 
Re spectrum key: The HDF viewer shows them as column vector, not in a horizontal row. But maybe this does not matter then.

Reply:
> Re spectrum key: The HDF viewer shows them as column vector, not in a horizontal row. But maybe this does not matter then.

All 1D dataset are shown as column. Whether it is a "row" or "column" is defined only with respect to other (e.g., 2D) arrays.